packages/web/lib/browser.js

```diff
@@ -8,20 +10,63 @@ const {
   resolveFn,
   resolveFnScope,
   resolveInternalScope,
+  validateScope,
 } = require("@browselang/core/lib/scope");
 const { help } = require("@browselang/core/lib/utils");
 const UrlPattern = require("url-pattern");
 const { BrowseError } = require("@browselang/core/lib/error");
 
 const getPageScope = require("./page");
 
+const throws = (fn) => (...args) => {
+  try {
+    return { success: true, value: fn(...args) };
+  } catch (e) {
+    return { success: false };
```

Comment:
We don't need it here, but you CAN actually add the `err` to the object if you want.

Just a thought for if/when we move this to a utility function